packages/client/src/checkout/types/index.ts

```diff
@@ -1,3 +1,6 @@
+export * from '../draftOrders/types/draftOrder.types.js';
```

Comment:
You could replace all these exports with:

```suggestion
export * from '../draftOrders/types';
```

Reply:
I can’t do 

`export * from '../draftOrders/types';`

but I changed to 

`export * from '../draftOrders/types/index.js';`

Reply:
My bad, I forgot about that restriction from nodenext :D
Yes, that was what I meant.